test_scripts/API/AutoCompleteList/008_ATF_SetGlobalProperties_P_AutoCompleteList_ArrayOfUpperBound.lua

```diff
@@ -0,0 +1,111 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
```

Comment:
@Aleksandriys, please add to requirement summary also req:
[SetGlobalProperties] SDL must transfer request to HMI in case of valid "autoCompleteList" param